corehq/apps/users/views/mobile/users.py

```diff
@@ -541,21 +541,21 @@ def check_username(self, in_data):
         except KeyError:
             return HttpResponseBadRequest('You must specify a username')
         if username == 'admin' or username == 'demo_user':
-            return {'error': _('Username {} is reserved.'.format(username))}
+            return {'error': _(u'Username {} is reserved.').format(username)}
```

Comment:
I think the correct fix would be to put the `.format` outside of the translation (otherwise the username will be part of the translation string and a translation match will never be found)